### ISSUE TITLE
How come no npm publish?

### ISSUE THREAD

**Issue by JohnMcLear** [words/wordnet#8]
Seems like the npm package is way out of date?

**Comment by zeke** [comment]
Huh. Looks like I'm an npm owner. I can publish a new version. brb

**Comment by zeke** [comment]
Wow. Eight years in the making. I published it as a new major version as I have no idea what the included changes are.

(Note: I didn't author this module, and I don't remember how I became an owner of it)

https://github.com/words/wordnet/releases/tag/v2.0.0

```
$ np major --no-2fa

		
Warning: No files field specified in package.json nor is a .npmignore file present. Having one of those will prevent you from accidentally publishing development-specific files along with your package's source code to npm.
		
? The following new files will be published the first time:
- db/data.adj
- db/data.adv
- db/data.noun
- db/data.verb
- db/index.adj
- db/index.adv
- db/index.noun
- db/index.verb
- examples/list.js
- examples/lookup.js
- lib/reader.js
- lib/wordnet.js
- test/fixture.json
- test/wordnet.js
Continue? Yes

Publish a new version of wordnet (current: 1.0.0)

Commits:
- Fix typo in readme  5d814bb
- Fix small typo in example code!  66bbd42
- Merge pull request #5 from words/cleanup  4d716d4
- Formatting.  3c7b49a
- Update npm package info and bump version.  f31f030
- Update examples and docs.  2ff39c9
- Allow skipping pointers when performing lookup.  882968b
- Ignore package lock.  db2a7db
- Remove the outdated example from README.md  416b44c
- Merge pull request #4 from hijonathan/db-source  7db9539
- Merge pull request #3 from hijonathan/update-2020  49c1493
- Allow pointing to separate database source.  e398862
- Fix race condition.  d24baef
- Add tests.  234fb7b
- Remove unused dep.  ab567b4
- Abstact key prefix to variable.  208d9ba
- Clean up, better documentation.  1bcc5db
- Refactor to use promises instead of callbacks.  e6bf1f9
- More descriptive npm package info.  1c7a854
- Fix look up for words with space character.  02bd01b
- Add list() method.  a2fc6d8
- Update package.json.  ae9a1dc
- Update README.md  462935a
- Include more lex stuff.  6b7636d
- Some error handling.  7b0c717
- Handle glossary-less entries.  6333fc1
- Display only syntactic pointers.  418b45f
- Handle pointers.  4554743
- Fix adjective synset type.  9cf095b
- Better example.  d8d380e
- Read data + more package info.  ea33bf2
- Initial data read.  f572d3b

Commit Range:
e660028cc62636fabb0058e8a74f2275628a8b41...master

Registry:
https://registry.npmjs.org/


  ✔ Prerequisite check
  ✔ Git
  ✔ Cleanup
  ✔ Installing dependencies using npm
  ✔ Running tests using npm
  ✔ Bumping version using npm
  ✔ Publishing package using npm
  ✔ Pushing tags
  ✔ Creating release draft on GitHub

 wordnet 2.0.0 published 🎉`
```
